.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off"
+    },
+    "settings": {
+        "import/resolver": {
+          "node": {
+            "extensions": [""]
+          }
+        }
+      }
```

Comment:
Do not edit the Eslint file ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)

Reply:
Done

Reply:
[Done](https://github.com/IvanDerlich/restaurant-homepage/pull/3/commits/5c0e0e71380a7924cc4cc06315c49e3246c3010c)